test/eventlistener_test.go

```diff
@@ -227,6 +227,20 @@ func TestEventListenerCreate(t *testing.T) {
 			),
 			bldr.EventListenerSpec(
 				bldr.EventListenerServiceAccount(sa.Name),
+				bldr.EventListenerPodTemplate(
+					bldr.EventListenerPodTemplateSpec(
+						bldr.EventListenerPodTemplateTolerations(nil),
+						// If you want to test toleration feature, please use this commented part.
```

Comment:
Same question as above...is this failing in CI? If it is and it is really hard to test this on our current e2e setup, we can open an issue for it. But I think I'd prefer not having commented out bits of code

Reply:
I left comment above! 👍